src/Robo/Commands/Saml/SimpleSamlPhpCommand.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+namespace Acquia\Blt\Robo\Commands\Saml;
+
+use Acquia\Blt\Robo\BltTasks;
+use Robo\Contract\VerbosityThresholdInterface;
+
+/**
+ * Defines commands in the "simplesamlphp:*" namespace.
+ */
+class SimpleSamlPhpCommand extends BltTasks {
+
+  protected $bltRoot;
+  protected $repoRoot;
+
+  /**
+   * This hook will fire for all commands in this command file.
+   *
+   * @hook init
+   */
+  public function initialize() {
+    $this->bltRoot = $this->getConfigValue('blt.root');
+    $this->repoRoot = $this->getConfigValue('repo.root');
+  }
+
+  /**
+   * Initializes SimpleSAMLphp for project.
+   *
+   * @return \Robo\Result
+   * @throws \Exception
+   *
+   * @command simplesamlphp:init
+   */
+  public function simpleSamlPhpInit() {
+    if (!$this->isSimpleSamlPhpInstalled()) {
+      $result = $this->simpleSamlPhpLibInit();
+      $result = $this->simpleSamlPhpConfigInit();
+      $result = $this->setSimpleSamlPhpInstalled();
+      $result = $this->symlinkDocrootToLibDir();
+    }
+    else {
+      $this->say('SimpleSAMLphp has already been initialized by BLT.');
+    }
+    $this->simpleSamlPhpInitComplete();
+    if (isset($result)) {
+      return $result;
+    }
+  }
+
+  /**
+   * Adds simplesamlphp_auth as a dependency.
+   *
+   * @return \Robo\Result
+   * @throws \Exception
+   */
+  protected function simpleSamlPhpLibInit() {
```

Comment:
The method names don't all need to start with simpleSamlPhp, since this file is dedicated to that namespace. I'd suggest something more descriptive like `requirePackages()` or `requireModule()`